cumulus_library/cli.py

```diff
@@ -297,6 +273,7 @@ def run_cli(args: Dict):
                 for target in args["target"]:
                     builder.export_study(study_dict[target], args["data_path"])
 
+        db_backend.close()
         # returning the builder for ease of unit testing
         return builder
```

Comment:
Depends how we want to do it. In practice, this ensures that the duckdb backing file is flushed to disk.

So in a test, best approach might be to open that file back up using duckdb and do queries there to inspect the final state?

Reply:
oh i do like writing to a temp file and reading it back for validation, that feels clean - but we'd then want to get that cleaned up before each test run.

I do like the [pytest fixture](https://github.com/smart-on-fhir/cumulus-aggregator/blob/main/tests/conftest.py) approach for this over in the aggregator, but i know that's not your fave pattern - open to alternates

Reply:
I'm only really adding one unit test in this PR (though it's a big end-to-end one that covers a lot of cases).

So I went with being unobtrusive and just did a little `with tempfile.TemporaryDirectory() as tmpdir:` and wrote a file there, which gets cleaned up automatically.

I can totes see a fixture making sense here to reduce boilerplate, but maybe if/once you use duckdb more extensively in the tests, it would be more obvious how best to write that.

Reply:
(In my unit test I never actually manually open the file, I just export counts and make sure they match what I expect.)